src/seq/nucleotide.jl

```diff
@@ -119,16 +119,27 @@ end
 # represented with an N mask BitArray. If the ns[i] bit is set, then the
 # sequence may have any nucleotide at that position and it must be ignored.
 #
-# Sequences are immutable by convention. Low-level functions will mutate the
-# underlying data, any user-facing function that does so should have `unsafe` in
-# the name. Immutability allows two sequences to share the same underlying data,
```

Comment:
I'm not sure if they are useful in this situation. In fact, I want to know why are they not useful/used?